### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by laurenzlong**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,9 @@
+- Enables functions runtime to be set in `firebase.json` (#2241, thanks @quentinvernot!), for example:
```
Comment (bkendall) [inline CHANGELOG.md:1]:
```suggestion
- Enables Firebase Functions runtime to be set in `firebase.json` (#2241, thanks @quentinvernot!), for example:
```

**Comment by bkendall** [comment]
It looks like you'll need to run the formatter for this file. `npm run format` should do it.

File: CHANGELOG.md (hunk 1 above)
Comment (mbleigh) [reply CHANGELOG.md:1]: Firebase Functions isn't a thing! Cloud Functions for Firebase

File: CHANGELOG.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+```
```
Comment (mbleigh) [inline CHANGELOG.md:2]: I think you want to indent this so it picks up as part of the bulleted list.

**Pull request merged**